src/static/js/views/faq.html

```diff
@@ -86,14 +92,14 @@ <h4>Frequently Asked Questions</h4>
         How accurate is the package information that Software Discovery Tool provides?
       </div>
       <div class="list-group-item" ng-show="questionState(9)">
-        <p>The package information is regularly updated. It is accurate as on the date of the update. However, the information might become obsolete between updates. To obtain the most current package information, go to the official Linux distribution websites.</p>
+        <p>The package information is regularly updated. It is accurate as on the date of the update. However, the information might become obsolete between updates. To obtain the most current package information, go to the respective official Operating System websites.</p>
       </div>
       <div class="list-group-item list-group-item-info ng-class:{'active': questionState(10), 'glyphicon-minus':questionState(10), 'glyphicon-plus':!questionState(10)};"  ng-click="clickQuestion(10)">
-        Where can I find more information on running Linux open source packages on IBM LinuxONE and IBM z Systems?
+        Where can I find more information on running open source packages on z/OS and other Linux distributions?
       </div>
       <div class="list-group-item" ng-show="questionState(10)">
-        <p>More information on open source packages that have been ported and/or validated on corresponding distro versions by IBM could be found on <a href="https://www.ibm.com/developerworks/community/forums/html/topic?id=5dee144a-7c64-4bfe-884f-751d6308dbdf">Forum</a> from <a href="https://www.ibm.com/developerworks/community/groups/community/lozopensource">Community</a>.</p>
+        <p>More information on open source packages that have been ported and/or validated on corresponding distro versions by IBM could be found on <a href="https://www.ibm.com/community/z/open-source-software/">This Table</a> from <a href=" https://www.ibm.com/community/z/open-source/">Community</a>.</p>
```

Comment:
It occurs to me that this is a pretty limited view, which made sense for PDS (which was the same team that created the above table), but less so for a general open source project. So maybe drop mention of that table altogether, and link to The Open Mainframe Project (openmainframeproject.org) and the "IBM Community for Open Source on IBM Z and LinuxONE" (use this text rather than just "Community")